src/main/java/seedu/address/model/Entity/Id.java

```diff
@@ -0,0 +1,103 @@
+package seedu.address.model.Entity;
+
+import java.util.Objects;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+public class Id {
+
+    private static final String TYPE_CHARACTERS = "PDIMT";
+
+    public static final String MESSAGE_CONSTRAINTS_INVALID_NUMBER = "Number should be of the format integer,"
+            + "and adhere to the following constraints: \n"
+            + "1. It should  number which can be any digit."
+            + "The number must: \n"
+            + "    -be at least one digit long\n"
+            + "    -contain only digits from 0 to 9\n";
+
+    public static final String MESSAGE_CONSTRAINTS_INVALID_PREFIX = "Prefix should adhere to the following constraints:\n"
```

Comment:
It seems like Joshua implemented a MESSAGE_CONSTRAINTS string for invalid prefixes within the Id.java class. Maybe you could place this in the PrefixType enum instead? It's reeally up to you, but could you update me on your decision with respect to this, because it affects my code a little. Thanks Joanna